README.md

```diff
@@ -38,6 +38,7 @@ GOPATH` to find out where your `GOPATH` is). The apps currently include:
   e.g. downloaded from TradingView. The CSV format is compatible with
   `parfait-list` output, and in tandem these two apps allow for manual editing
   of the DB contents.
+- [parfait-screener] - generate a list of stocks that satisfy search criteria.
```

Comment:
"generating" (make consistent). Or, better yet, convert everything to present tense, "generate" etc.